src/backend/distributed/utils/shard_utils.c

```diff
@@ -36,3 +37,21 @@ GetTableLocalShardOid(Oid citusTableOid, uint64 shardId)
 
 	return shardRelationOid;
 }
+
+
+/*
+ * GetLongestShardName is a utility function that returns the name of the shard of a
+ * table that has the longest name in terms of number of characters.
+ *
+ * Both the Oid and name of the table are required so we can create longest shard names
+ * after a RENAME.
+ */
+char *
+GetLongestShardName(Oid citusTableOid, char *citusTableName)
```

Comment:
Again, for this function to get both the `citusTableOid ` and `citusTableName ` is confusing as one normally would expect `citusTableName= get_rel_name(citusTableOid) `. Can we call this `finalCitusTableName` or such?